torch/csrc/jit/init.cpp

```diff
@@ -61,6 +86,22 @@ void initJITBindings(PyObject *module) {
      return py::reinterpret_steal<py::object>(python::unflatten(vars, desc));
    });
 
+   py::class_<GraphExecutor>(m,"GraphExecutor")
+   .def(py::init([](py::function func, std::vector<tracer::TraceInput> inputs){
+     return createExecutorByTracing(true, func, std::move(inputs));
+   }))
+   .def("__call__",[](py::args args) {
+     auto ge = py::cast<GraphExecutor&>(args[0]);
+     auto inputs = py::cast<std::vector<at::Tensor>>(tuple_tail(args));
```

Comment:
A comment saying why do we have to do this instead of letting pybind parse the arguments would be helpful (I still remember, but who knows when we'll forget)

Reply:
I have to double-check this. But I think we updated to post-bug-fix and can remove this. When I started, that patch had not landed yet.